analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -207,10 +207,10 @@ class Maven(config: AnalyzerConfiguration) : PackageManager(config) {
                 localRepositoryManager.add(session, request)
 
         override fun find(session: RepositorySystemSession, request: LocalArtifactRequest): LocalArtifactResult {
-            log.debug { "Request to find local artifact: $request" }
+            val id = request.artifact.identifier()
+            projectsByIdentifier[id]?.let {
+                log.debug { "Artifact '$id' refers to the local project in '${it.pomFile}'." }
```

Comment:
This message lacks context, it should somehow explain that the artifact request to the local repository is forwarded to the POM file (which is not in the local repository), because it is part of the same multi-module project. While I used "local project" in code I think it is not clear to the user what it means here.

Reply:
I changed the wording, hoping it is better now.